data/duke.txt

```diff
@@ -0,0 +1,2 @@
+D | 1 | Submit report | 2023-09-01 1800
+E | 0 | Team meeting | 2023-01-01 1300 | 2023-09-02 1200
```

Comment:
Consider adding the file to `.gitignore`. The data file is not relevant to be pushed to the respository!